python/ray/experimental/state/state_manager.py

```diff
@@ -440,11 +440,7 @@ async def stream_log(
             ),
             timeout=timeout,
         )
-        await self._validate_stream(stream)
-        return stream
-
-    @staticmethod
-    async def _validate_stream(stream):
         metadata = await stream.initial_metadata()
         if metadata.get(log_consts.LOG_GRPC_ERROR) == log_consts.FILE_NOT_FOUND:
-            raise ValueError('File "{log_file_name}" not found on node {node_id}')
+            raise ValueError(f'File "{log_file_name}" not found on node {node_id}')
+        return stream
```

Comment:
Does it have meaning to return here?

Reply:
Yes? Because I removed the `_validate_stream` func. this is the same return as before.

Reply:
oh yeah I was confused that this raise always happens...